src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC, useContext, useEffect } from 'react';
+
+import { errors } from '../../constants';
+
+import { AppContext } from '../../wrappers/AppProvider';
+
+export const ErrorNotification: FC = () => {
+  const { errorType, setErrorType } = useContext(AppContext);
+
+  const errorMessage = errorType ? errors[errorType].message : '';
+
+  useEffect(() => {
+    if (errorType) {
+      setTimeout(() => {
+        setErrorType(null);
+      }, 3000);
+    }
+  });
```

Comment:
It makes sense to add `errorType` as the dependency of `useEffect` in this case: there's no need to call this effect on every render, only when `errorType` changes.
Also it makes sense to clear the timeout before `errorType` changes:
```
useEffect(() => {
  // set timeout
  return () => {
    // clear timeout
  };
}, [errorType]);
```